### ISSUE TITLE
begin and commit keywords cause a 500 error

### ISSUE THREAD

**Issue by hoxtalicious** [phpmyadmin/phpmyadmin#13240]
### Steps to reproduce
Try and do a statement like
```
begin;
SELECT * FROM `tablename`
commit;
```
on any table. Here is a [video demonstrating the error](https://www.youtube.com/watch?v=ahIqnuSAbkM)

### Server configuration

**Server Details:**
At the time, the demo server was running nginx/1.10.3
demo.phpmyadmin.com (although I initially discovered the misbehaviour on my own server)

**Database:**
world.Country

**PHP version:**
7.0.16-3

**phpMyAdmin version:**
Tested and present on 4.8.0-dev and 4.7.0

### Client configuration
**Browser:**
Chrome v58.0.3029.81 (64-bit)

**Operating system:**
Fedora Workstation 25

**Comment by nijel** [comment]
Bug in SQL parser:

```
NOTICE: PHP message: PHP Fatal error:  Uncaught Error: Call to a member function has() on null in phpmyadmin/vendor/phpmyadmin/sql-parser/src/Statements/TransactionStatement.php:85
Stack trace:
#0 phpmyadmin/vendor/phpmyadmin/sql-parser/src/Statement.php(90): PhpMyAdmin\SqlParser\Statements\TransactionStatement->parse(Object(PhpMyAdmin\SqlParser\Parser), Object(PhpMyAdmin\SqlParser\TokensList))
#1 phpmyadmin/vendor/phpmyadmin/sql-parser/src/Parser.php(464): PhpMyAdmin\SqlParser\Statement->__construct(Object(PhpMyAdmin\SqlParser\Parser), Object(PhpMyAdmin\SqlParser\TokensList))
#2 phpmyadmin/vendor/phpmyadmin/sql-parser/src/Parser.php(345): PhpMyAdmin\SqlParser\Parser->parse()
#3 phpmyadmin/libraries/Linter.php(123): PhpMyAdmin\SqlParser\Parser->__construct(Object(PhpMyAdmin\SqlParser\TokensList))
#4 phpmyadmin/lint.php(50): PMA\libraries\Linter::lint('begin;\nSELECT *.....
```

**Comment by nijel** [comment]
I've just released SQL Parser 4.1.4 with the fix.

**Comment by hoxtalicious** [comment]
Yep, I was actually getting full trace on my server but not on the demo server. Great you've released a fix.
